streams/src/test/java/org/apache/kafka/streams/integration/GlobalKTableEOSIntegrationTest.java

```diff
@@ -339,17 +446,17 @@ public void shouldNotRestoreAbortedMessages() throws Exception {
             .getStore(globalStore, kafkaStreams, QueryableStoreTypes.keyValueStore());
         assertNotNull(store);
 
-        final Map<Long, String> result = new HashMap<>();
+        final Map<Long, String> storeContent = new HashMap<>();
         TestUtils.waitForCondition(
             () -> {
-                result.clear();
-                store.all().forEachRemaining(pair -> result.put(pair.key, pair.value));
-                return result.equals(expected);
+                storeContent.clear();
+                store.all().forEachRemaining(pair -> storeContent.put(pair.key, pair.value));
+                return storeContent.equals(expected);
             },
-            30000L,
+            30_000L,
             () -> "waiting for initial values" +
                 "\n  expected: " + expected +
-                "\n  received: " + results
```

Comment:
side fix: this should have be `result` (singular) not `results` plural to refer to the local variable instead of the global one. Renaming to avoid this issue in the future.